src/octopeer-github/main/ElementEventBinding/ScrollIntoViewElementEventBinding.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by Mathias on 2016-05-11.
+ * Class to create ScrollIntoView events
+ */
+class ScrollIntoViewElementEventBinding implements ElementEventBinding {
+    public eventType: string = "scrollintoview";
```

Comment:
We might be able to fix this with some viewport manipulation. We can also leave it out. Was added for completeness.